src/azure-web-command-service.ts

```diff
@@ -0,0 +1,77 @@
+import { Guid } from "guid-typescript";
+import { LocalStorageItem } from "./azure-web-local-storage-item";
+
+/**
+ * Manage local stored azure web event commands.
+ */
+export class AzureWebCommandService {
+    private static localStorageKey: string = "SENDMESSAGES";
+
+    /**
+     * Get a stored command by its id.
+     * @param commandId The command id.
+     * @returns If found the command id or null if not found.
+     */
+    public static getCommand(commandId: Guid): LocalStorageItem | null {
```

Comment:
Any reason we use null instead of undefined? With undefined we can make our lives easier by using stuff like the ?. propagation operator etc.

Reply:
I preferred null for me because the value was not present instead of not declared. Changed for the reasons you gave.